### PR TITLE
Change maxLength for create_issue

### LINKED ISSUE #411: Allow more characters in the title field when creating a Github Issue from MM
Currently it stops you from entering more than ~45 characters I think....annoying.

### PR DESCRIPTION
**Pull request by haardikdharma10**
#### Summary

This PR aims to solve #411 

#### Ticket Link

Fixes #411

### CODE REVIEW

File: webapp/src/components/modals/create_issue/create_issue.jsx [hunk 1]
```diff
@@ -189,7 +189,7 @@ export default class CreateIssueModal extends PureComponent {
                     type='input'
                     required={true}
                     disabled={false}
-                    maxLength={65}
+                    maxLength={120}
```
Comment (hanzei) [inline webapp/src/components/modals/create_issue/create_issue.jsx:192]: Have you tested the impact of this change? Just want to double check that the GH API accepts this.

**Review by hanzei — CHANGES_REQUESTED** [review]
Thanks for the PR. One concern before this LGTM

File: webapp/src/components/modals/create_issue/create_issue.jsx (hunk 1 above)
Comment (haardikdharma10) [reply webapp/src/components/modals/create_issue/create_issue.jsx:192]: Hi @hanzei, thanks for the review. No issues found. Perhaps, a QA review can confirm:)

**Review by hanzei — CHANGES_REQUESTED** [review]
LGTM

**Review by hanzei — APPROVED** [review]
Meant to approve

**Review by larkox — APPROVED** [review]

**Comment by haardikdharma10** [comment]
Heads up on this one. 
cc @jasonblais @jfrerich @emilyacook @hanzei

**Comment by hanzei** [comment]
@DHaussermann Could you please do QA review for this ticket when you have the time?

**Comment by DHaussermann** [comment]
@haardikdharma10 Thanks for this PR! Sorry for the delay in testing this.

This change certainly improves the usability with the higher character limit. I do see a small gap here. The limit is enforced as expected when typing the characters or pasting from the clipboard into the box. However this same limit is nor enforced when creating via slash command. `/github issue create` In this case the post text which can exceed the limit is already in the box so, the validation is circumvented.      
image
 
0/5 This may be slightly out of scope but are you able to trim the text when this slash command is used similar to how the text is truncated when pasting from the clipboard?
Please let me know if this would be better addressed in a different PR as it seems to be a preexisting issue.

File: webapp/src/components/modals/create_issue/create_issue.jsx (hunk 1 above)
Comment (mickmister) [inline webapp/src/components/modals/create_issue/create_issue.jsx:192]: Hi @haardikdharma10, can we change this to GitHub's limit of 256?

Comment (haardikdharma10) [reply webapp/src/components/modals/create_issue/create_issue.jsx]: Sure thing @mickmister!

**Review by haardikdharma10 — COMMENTED** [review]
@DHaussermann hmmm... Can we open a separate ticket for the issue you are talking about as you said, seems to be a pre-existing issue.

**Comment by mickmister** [comment]
Maybe we can just truncate the string to 256 characters for these two cases here:

https://github.com/mattermost/mattermost-plugin-github/blob/b7c775599ac29be86c219f544392ba7e93e1d713/webapp/src/components/modals/create_issue/create_issue.jsx#L46-L52

**Comment by haardikdharma10** [comment]
Done! @mickmister

File: webapp/src/components/modals/create_issue/create_issue.jsx [hunk 2]
```diff
@@ -189,7 +192,7 @@ export default class CreateIssueModal extends PureComponent {
                     type='input'
                     required={true}
                     disabled={false}
-                    maxLength={65}
+                    maxLength={256}
```
Comment (mickmister) [inline webapp/src/components/modals/create_issue/create_issue.jsx:195]:
```suggestion
                    maxLength={MAX_TITLE_LENGTH}
```

**Review by mickmister — APPROVED** [review]
LGTM! Left one improvement to reuse the constant added

**Comment by haardikdharma10** [comment]
@DHaussermann ping for a review:)

**Review by DHaussermann — APPROVED** [review]
Tested and passed
In addition to previous test this now also takes the forst part of a post when the character limit is exceeded.
LGTM!


Huge thank @haardikdharma10 This is a great UX improvement. I appreciate your continued efforts on this.

**Comment by haardikdharma10** [comment]
Thank you @DHaussermann! Good enough to merge? 

c.c @mickmister

**Pull request merged**